modules/cluster-logging-release-notes-5.3.z.adoc

```diff
@@ -1,5 +1,29 @@
 //Z-stream Release Notes by Version
 
+[id="cluster-logging-release-notes-5-3-5"]
+= OpenShift Logging 5.3.5
+This release includes link:https://access.redhat.com/errata/RHSA-2022:0721[RHSA-2022:0721 OpenShift Logging Bug Fix Release 5.3.5]
```

Comment:
```suggestion
[role="_abstract"]
This release includes link:https://access.redhat.com/errata/RHSA-2022:0721[RHSA-2022:0721 OpenShift Logging Bug Fix Release 5.3.5]
```
I think you need to add a line before the first sentence to mark it as an abstract

Reply:
Fixed, thanks!